.github/workflows/demo-contact-diary-app.yml

```diff
@@ -0,0 +1,41 @@
+# https://docs.spring.io/spring-boot/docs/current/maven-plugin/reference/htmlsingle/#build-image
+
+# Build and package image for demo-contact-diary-app and deploy it to docker.io
+name: demo-contact-diary-app Develop
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
With version 3 I get the following error (docker-compose 1.25.0):
```
ERROR: The Compose file './docker-compose-postgres.yml' is invalid because:
services.postgres.healthcheck value Additional properties are not allowed ('start_period' was unexpected)
```
The start_period parameter seems to be removed in version 3. With 2.3 it works.

Reply:
Fixed.